vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_processor.cc

```diff
@@ -0,0 +1,129 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_processor.h"
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_events.h"
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_processor_util.h"
```

Comment:
these two includes should not be part of the main include

Reply:
Removing these includes results in errors

Reply:
you need to move them down to be with the other includes as there should only be a single include at the top of the file which reflects the interface for that implementation

Reply:
Ah that's what you meant, will do